sdcontent/INSTRUCTIONS-README.txt

```diff
@@ -1,34 +1,38 @@
 NOOBS INSTALLATION INSTRUCTIONS
 
-1. Insert an SD card that is 8GB or greater in size into your computer.
+1. Insert an SD card that is 16GB or greater in size into your computer.
 2. Format the SD card using the platform-specific instructions below:
    a. Windows (32GB cards and under)
       i. Download the SD Association's Formatting Tool from https://www.sdcard.org/downloads/formatter_4/eula_windows/
       ii. Install and run the Formatting Tool on your machine
-      iii. Check that the SD card you inserted matches the one selected by the Tool
+      iii. Check that the SD card you inserted matches the one selected by the Formatting Tool
       iv. Click the "Format" button
    b. Mac (32GB cards and under)
       i. Download the SD Association's Formatting Tool from https://www.sdcard.org/downloads/formatter_4/eula_mac/
       ii. Install and run the Formatting Tool on your machine
       iii. Select "Overwrite Format"
-      iv. Check that the SD card you inserted matches the one selected by the Tool
+      iv. Check that the SD card you inserted matches the one selected by the tool

```

Comment:
Sorry to nit-pick, but I think the language here should match line 8 i.e. "tool" -> "Formatting Tool".